docs/pages/component/props.mdx

```diff
@@ -47,14 +47,32 @@ A Boolean value that indicates whether the player should automatically delay pla
 - **false** - Immediately starts playback
 - **true (default)** - Delays playback in order to minimize stalling
 
+### `controlsStyles`
+
+<PlatformsList types={['Android']} />
+
+Adjust the control styles. This prop takes an object with one property listed below.
+
+| Property    | Type    | Description                                                                          |
+|-------------|---------|--------------------------------------------------------------------------------------|
+| hideSeekBar | boolean | The default value is `false`, allowing you to hide the seek bar for live broadcasts. |
+
+Example with default values:
+
+```javascript
+controlsStyles={{
+  hideSeekBar: false,
+}}
+```
+
 ### `bufferConfig`
 
 <PlatformsList types={['Android']} />
 
 Adjust the buffer settings. This prop takes an object with one or more of the properties listed below.
 
 | Property                          | Type   | Description                                                                                                                                                                                     |
-| --------------------------------- | ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|-----------------------------------|--------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
same here : can you remove these patch please ?
No need that you become owner of these line !